### DISCUSSION TITLE
Numpy doesn't support Silicon Mac hardware

### DISCUSSION THREAD

**Discussion by TheMissingPort** [UltimaHoarder/UltimaScraper#1115] [General]
When I try pip3 install Numpy I get:
ERROR: Could not build wheels for numpy which use PEP 517 and cannot be installed directly

After digging through Numpy on GitHub, there doesn't seem to be much solution, or atleast requires a lot more tinkering, but I haven't been able to get it working

Any suggestions? Thanks

**Comment by UltimaHoarder** [comment]
Remove numpy from the requirements
